go/service/config.go

```diff
@@ -161,6 +161,38 @@ func (h ConfigHandler) SetUserConfig(_ context.Context, arg keybase1.SetUserConf
 	return nil
 }
 
+func (h ConfigHandler) SetPath(_ context.Context, arg keybase1.SetPathArg) error {
```

Comment:
so this is going to happen every time a client connects?  but it will be idempotent?  i guess an issue s that a user can never delete something from his path and have it reflected here.